.github/workflows/swift-toolchain.yml

```diff
@@ -55,6 +55,7 @@ on:
 
 env:
   SCCACHE_DIRECT: yes
+  ANDROID_API_LEVEL: 28
```

Comment:
Do we have the space to move this into an input? That would make this configurable. I'm not a fan of using the environment to propagate settings.